actionpack/test/controller/integration_test.rb

```diff
@@ -170,9 +170,12 @@ def method_missing(name, *args)
 class IntegrationTestUsesCorrectClass < ActionDispatch::IntegrationTest
   def test_integration_methods_called
     reset!
+    headers = { "Origin" => "*" }
 
-    %w( get post head patch put delete ).each do |verb|
-      assert_nothing_raised { __send__(verb, "/") }
+    %w( get post head patch put delete options ).each do |verb|
+      assert_silent do
+        assert_nothing_raised { __send__(verb, "/", headers: headers) }
```

Comment:
Can you remove the `assert_silent` only for Ruby 2.7? Now this test will be failed with Ruby master so we can easily find any regression without `assert_silent`.

Reply:
Updated!